### PR TITLE
fix(DB/creature_loot_template): Pattern: Fur Lining - Arcane Resist

### PR DESCRIPTION
**Pull request by Si1ker**
This change will add the item to the NPC Cult Researcher.

Reference: https://db.rising-gods.de/?item=44563#dropped-by

## CHANGES PROPOSED:
-  Add the item Pattern: Fur Lining - Arcane Resist to the NPC Cult Researcher as a possible loot

## ISSUES ADDRESSED:
- Never reported

## TESTS PERFORMED:
- Applied SQL change
- Tested in-game

image

## HOW TO TEST THE CHANGES:

**Before applying the SQL changes:**
- .go c id 32297
- Kill A LOT of Cult Researchers
- The recipe will never be a drop from them

**After applying the SQL changes:**
- .go c id 32297
[note: You will need to learn Leatherworking at high level in order to see this item]
- .learn 51302
- .setskill 165 450
- Kill a lot of Cult Researchers (remember it has a prop of 1%)

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1601361978416709800.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601361978416709800');
+
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+(32297, 44563, 0, 1, 0, 1, 0, 1, 1); -- reference: https://db.rising-gods.de/?item=44563#dropped-by
+
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601361978416709800.sql:5]:
```suggestion
DELETE FROM `creature_loot_template` WHERE `entry` = 32297 AND `item` = 44563;
INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
(32297, 44563, 0, 1, 0, 1, 0, 1, 1); -- reference: https://db.rising-gods.de/?item=44563#dropped-by
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Always DELETE before INSERT

**Comment by Kitzunu** [comment]
Also make sure  Frostbrood Spawn drops this item. src https://wow.gamepedia.com/Pattern:_Fur_Lining_-_Arcane_Resist

**Comment by Si1ker** [comment]
@Kitzunu Before this PR, the item is not a drop.. literally anywhere.

About the NPC "Frostbrood Spawn", it does not use/drop this item.

This is its loot:
image

Changes done ! 🐱

File: data/sql/updates/pending_db_world/rev_1601361978416709800.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601361978416709800');
+
+DELETE FROM `creature_loot_template` WHERE `entry` = 32297 AND `item` = 44563;
+DELETE FROM `creature_loot_template` WHERE `entry` = 31702 AND `item` = 44563;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+(32297, 44563, 0, 1, 0, 1, 0, 1, 1); -- Cult Researcher: https://db.rising-gods.de/?item=44563#dropped-by
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601361978416709800.sql:6]:
```suggestion
(32297, 44563, 0, 1, 0, 1, 0, 1, 1), -- Cult Researcher: https://db.rising-gods.de/?item=44563#dropped-by
```

File: data/sql/updates/pending_db_world/rev_1601361978416709800.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601361978416709800');
+
+DELETE FROM `creature_loot_template` WHERE `entry` = 32297 AND `item` = 44563;
+DELETE FROM `creature_loot_template` WHERE `entry` = 31702 AND `item` = 44563;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601361978416709800.sql:4]:
```suggestion
DELETE FROM `creature_loot_template` WHERE `entry` IN (31702, 32297) AND `item` = 44563;
```

File: data/sql/updates/pending_db_world/rev_1601361978416709800.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601361978416709800');
+
+DELETE FROM `creature_loot_template` WHERE `entry` = 32297 AND `item` = 44563;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601361978416709800.sql:3]:
```suggestion
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by Si1ker** [comment]
@Kitzunu 🐱

**Review by Kitzunu — APPROVED** [review]

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by wizzymore** [comment]
Hi,

The item now drops, i only got it to drop once on the Cult Researchers as they are easier to farm, as the queries are the same, both are good.

Confirming that both npcs should drop it with a very low chance ( if it is 1% or less, only god knows ):
https://www.wowhead.com/item=44563/pattern-fur-lining-arcane-resist#comments:id=579568
https://www.wowhead.com/item=44563/pattern-fur-lining-arcane-resist#comments:id=587402

Tried to look for the drop chance on wowhead but it goes only to 2015 where it is still lower than it was on WotLK so we should not count on this
http://web.archive.org/web/20150630004928/https://www.wowhead.com/item=44563/pattern-fur-lining-arcane-resist

```

Cult Researcher<Cult of the Damned> 0.19
Frostbrood Spawn 0.06
```

**Comment by Si1ker** [comment]
@wizzymore Hello ! , thanks for testing and the link provided.
I have changed the rate values to the ones in 2015 because if you check the values nowadays, they were decreased by a minimal difference. Also, this is better than 1, both rates between 5 years of difference is not a big deal so we can keep it as are the oldest we can get, there is no evidence from 11 years ago so it's okay. 🐱 

Check Wowhead comments about this item, it was an epic moment to get on of these, there are people who never get one in all retail history.

2015
Cult Researcher<Cult of the Damned> 0.19
Frostbrood Spawn 0.06

2020
Cult Researcher<Cult of the Damned> 0.16
Frostbrood Spawn 0.05

**Pull request merged**